src/.eslintrc.cjs

```diff
@@ -22,52 +23,54 @@ module.exports = {
         "quotes": [ "error", "double" ],
         "semi": [ "error", "always" ],
         "camelcase": "off",
-        // "no-unused-vars": "off",
         "eqeqeq": "off",
         "comma-dangle": [ "error", "never" ],
         "comma-style": [ "error", "last" ],
         "comma-spacing": "off",
-        "space-before-function-paren": [ "error", "never" ],
         "space-in-parens": [ "error", "always" ],
-        "keyword-spacing": [ "error", {
-            "overrides": {
-                "if": { "before": false, "after": false },
-                "else": { "before": true, "after": true },
-                "for": { "before": false, "after": false },
-                "while": { "before": false, "after": false }
-            }
-        } ],
         "space-before-blocks": [ "error", "always" ],
         "array-bracket-spacing": [ "error", "always" ],
         "object-curly-spacing": [ "error", "always" ],
         "space-unary-ops": "off",
         "spaced-comment": "off",
         "curly": [ "error", "multi-or-nest" ],
         "nonblock-statement-body-position": [ "error", "below" ],
-        "one-var": "off",
         "no-unneeded-ternary": "off",
         "no-cond-assign": [ "error", "always" ],
-        "no-console": "off",
-        "new-cap": "off",
         "no-tabs": "off",
         "no-mixed-spaces-and-tabs": "off",
-        "no-prototype-builtins": "off",
-        "quote-props": "off",
-        "no-undef": "off",
         "no-useless-return": "off",
         "no-new": "off",
-        "no-useless-constructor": "off",
-        "no-lone-blocks": "off",
         "no-fallthrough": "off",
```

Comment:
Remove this rule, please

Reply:
removed `"no-useless-constructor": "off"` and `"no-lone-blocks": "off"` rules

Reply:
removed `"no-fallthrough": "off"` rule